src/components/Alert.vue

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+import store from '@/store';
+import {
+  computed, Ref, ref, watchEffect,
+} from '@vue/composition-api';
+import api from '@/api';
+
+export default {
+  name: 'Alert',
+
+  setup() {
+    const loadError = computed(() => store.getters.loadError);
+
+    // Vars to store the selected choices in
+    const workspace: Ref<string | null> = ref(null);
+    const network: Ref<string | null> = ref(null);
+
+    // Compute the workspace/network options
+    const workspaceOptions: Ref<string[]> = ref([]);
+    watchEffect(async () => {
+      workspaceOptions.value = await api.workspaces();
+    });
+
+    const networkOptions: Ref<string[]> = ref([]);
+    watchEffect(async () => {
+      if (workspace.value !== null) {
+        networkOptions.value = await api.graphs(workspace.value);
+      }
+    });
+
+    const buttonHref: Ref<string> = ref(loadError.value.href);
+    const buttonText: Ref<string> = ref('');
+    watchEffect(async () => {
+      if (workspace.value !== null && network.value !== null) {
+        buttonHref.value = `./?workspace=${workspace.value}&graph=${network.value}`;
+        buttonText.value = 'Go To Network';
+      } else if (loadError.value.message === 'There was a network issue when getting data') {
+        buttonHref.value = loadError.value.href;
+        buttonText.value = 'Refresh the page';
+      } else {
+        buttonHref.value = loadError.value.href;
+        buttonText.value = 'Back to MultiNet';
+      }
+    });
+
+    return {
+      buttonHref,
+      buttonText,
+      loadError,
+      network,
+      networkOptions,
+      workspace,
+      workspaceOptions,
+    };
+  },
+};
+</script>
+
+<template>
+  <div>
+    <v-alert
+      type="error"
```

Comment:
```suggestion
      type="warning"
      outlined
      text
      border="left"
```

Roni mentioned in his comment that a softer message would be better and I think a softer color for the alert would probably be better as well.